setup.py

```diff
@@ -3,17 +3,25 @@
 import os.path
 
 package_name = "dbt"
-package_version = "0.1.0-SNAPSHOT"
+package_version = "0.1.0"
 
 setup(
   name=package_name,
   version=package_version,
+  description="Data build tool for Analyst Collective",
+  author="Analyst Collective",
+  author_email="admin@analystcollective.org",
+  url="https://github.com/analyst-collective/dbt",
   packages=find_packages(),
+  entry_points={
+    'console_scripts': [
+      'dbt = dbt.main:main',
+    ],
+  },
   scripts=[
     'scripts/dbt',
   ],
   install_requires=[
-    'argparse>=1.2.1',
```

Comment:
@cmerrick I think argparse is built-in for python >= 2.7. I only removed it because there were problems `pip install`ing it on my machine... I can reinvestigate though